### PR TITLE
Add AnyTest package

### PR DESCRIPTION
**Pull request by timfjord**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

The main idea of this package is to automatically detect a test framework for the given file and run it.
Basically it is a Sublime Text interpretation of the [vim-test](https://github.com/vim-test/vim-test) Vim plugin.
Currently it supports only 7 test frameworks(`espec`, `exunit`, `jest`, `pytest`, `pyunit`, `minitest`, `rspec`)  but more test frameworks are coming very soon.

There might be some packages for individual test frameworks(like [TestRSpec](https://packagecontrol.io/packages/TestRSpec), etc) but the idea of `AnyTest` is to handle as many test framworks as possible in one pacakge.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by braver** [comment]
Looks interesting! Just because I feel like I need to nitpick on something, I'd like to see a setting that disables the context menu entries. Maybe that's just a pet peeve, but that area does get crowded. Or maybe remove the separator?

**Comment by braver** [comment]
Cool 👌🏻
